docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -1,3 +1,439 @@
+[Kafka Channel ConfigMap](../configuration/kafka-channel-configuration.md#create-a-kafka-channel-configmap) documentation.
+
+### Changing the default Channel implementation for a namespace
+
+You can modify the default Broker creation behavior for one or more namespaces.
+
+For example, if you wanted to use the `kafka-channel` ConfigMap for all other Brokers created, but wanted to use `config-br-default-channel` ConfigMap for `namespace-1` and `namespace-2`, you would use the following ConfigMap settings:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+      apiVersion: v1
+      kind: ConfigMap
+      name: config-kafka-channel
+      namespace: knative-eventing
+    namespaceDefaults:
+      namespace-1:
+        apiVersion: v1
+        kind: ConfigMap
+        name: config-br-default-channel
+        namespace: knative-eventing
+      namespace-2:
+        apiVersion: v1
+        kind: ConfigMap
+        name: config-br-default-channel
+        namespace: knative-eventing
+```
+
+## Configuring delivery spec defaults
+
+You can configure default event delivery parameters for Brokers that are applied in cases where an event fails to be delivered:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+      apiVersion: v1
+      kind: ConfigMap
+      name: config-kafka-channel
+      namespace: knative-eventing
+      delivery:
+        retry: 10
+        backoffDelay: PT0.2S
+        backoffPolicy: exponential
+    namespaceDefaults:
+      namespace-1:
+        apiVersion: v1
+        kind: ConfigMap
+        name: config-br-default-channel
+        namespace: knative-eventing
+        delivery:
+          deadLetterSink:
+            ref:
+              kind: Service
+              namespace: example-namespace
+              name: example-service
+              apiVersion: v1
+            uri: example-uri
+          retry: 10
+          backoffPolicy: exponential
+          backoffDelay: "PT0.2S"
+```
+
+### Dead letter sink
+
+You can configure the `deadLetterSink` delivery parameter so that if an event fails to be delivered it is sent to the specified event sink.
+
+### Retries
+
+You can set a minimum number of times that the delivery must be retried before the event is sent to the dead letter sink, by configuring the `retry` delivery parameter with an integer value.
```

Comment:
I am not sure how this all relates to Istio?

Reply:
It doesn't, but to put the Istio stuff here I needed to make the admin config section, so all of this stuff is just being moved over. You can ignore it for now since I just moved it, I didn't make any changes 🙂